tests/unit/labeller.spec.ts

```diff
@@ -649,6 +649,19 @@ describe('Labeller', () => {
     expect(hrl(step)).toEqual('Compute waterfall of "value" from "2019" to "2020"');
   });
 
+  it('generates label for totals steps', () => {
+    const step: S.AddTotalRowsStep = {
+      name: 'totals',
+      totals: [
+        ['COUNTRY', 'All countries'],
+        ['PRODUCT', 'All products'],
+      ],
+      aggregations: [{ columns: ['VALUE'], newcolumns: ['VALUE'], aggfunction: 'sum' }],
+      groups: ['DATE'],
+    };
+    expect(hrl(step)).toEqual('Add total rows in "COUNTRY", "PRODUCT"');
```

Comment:
```suggestion
    expect(hrl(step)).toEqual('Add total rows for "COUNTRY", "PRODUCT"');
```
? (not sure)

Reply:
I've opted for: `'Add total rows in columns "COUNTRY", "PRODUCT"'`